style.css

```diff
@@ -0,0 +1,849 @@
+  .list3 li {
+    float: left;
+    position: relative;
+    top: 10px;
+    left: 130px;
+    padding: 10px;
+  }
+
+  .main_image-one {
+    height: 287.8px;
+    grid-area: first;
+    margin-bottom: 0;
+    width: 498.66px;
+  }
+
+  .main_image-two {
+    grid-area: second;
+    margin-bottom: 0;
+    width: 245.33px;
+    height: 162px;
+  }
+
+  .main_image-three {
+    grid-area: third;
+    margin-bottom: 0;
+    width: 245.33px;
+    height: 119px;
+  }
+
+  .main_image {
+    display: grid;
+    grid-gap: 8px;
+    grid-template-areas:
+      "first second"
+      "first third";
+    grid-template-columns: 2fr 1fr;
+    margin-bottom: 0;
+  }
+
+  .details1 {
+    position: relative;
+    top: 115px;
+    font-size: 15px;
+  }
+
+  .main_image-one h2 {
+    position: relative;
+    top: 110px;
+    font-size: 30px;
+  }
+
+  .main_image-one h5 {
+    position: relative;
+    top: 110px;
+  }
+
+  .details2 {
+    position: relative;
+    top: 15px;
+  }
+
+  .main_image-two h2 {
+    font-size: 27px;
+    position: relative;
+    top: 6px;
+  }
+
+  .main_image-three h5 {
+    position: relative;
+    top: 1px;
+  }
+
+  .main_image-three h2 {
+    font-size: 18px;
+  }
+
+  .wrapper {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    margin-bottom: 10px;
+    grid-row-gap: 20px;
+  }
+
+  .latest_news,
+  .funds {
+    margin-top: 35px;
+  }
+
+  .news_article > img {
+    width: 229.33px;
+    height: 114.66px;
+  }
+
+  .news_article,
+  .news_deals {
+    display: block;
+  }
+
+  .head {
+    display: grid;
+    grid-column: 1 / span 3;
+    grid-template-columns: 1fr, 1fr;
+  }
+
+  .box-wrapper {
+    display: grid;
+    width: 97%;
+    grid-template-columns: repeat(3, 1fr);
+    grid-gap: 10px;
+    margin: auto;
+  }
+
+  .funds > div:last-child {
+    display: none;
+  }
+
+  .box {
+    width: 100%;
+    margin: auto;
+  }
+
+  .most-popular {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    margin-top: 50px;
+  }
+
+  .text-block h5 {
+    position: relative;
+    top: 10px;
+  }
+
+  .text-block h2 {
+    position: relative;
+    top: 1px;
+  }
+
+  .details5 {
+    position: relative;
+    bottom: 5px;
+  }
+
+  .news_deals > img {
+    width: 229.33px;
+    height: 170.84px;
+  }
+
+  .titles p {
+    width: 94.28px;
+    height: 18px;
+    margin-right: 10px;
+  }
+
+  .upper-footer {
+    width: 768px;
+    height: 161px;
+  }
+
+  .social-media li {
+    margin: 8px;
+    position: relative;
+    bottom: 20px;
+  }
+
+  .menu-links li {
+    margin: 1px;
+    position: relative;
+    bottom: 48px;
+  }
+
+  .lower-footer {
+    width: 768px;
+    height: 97px;
+  }
+}
+
+@media only screen and (min-width: 1024px) {
+  header img {
+    display: none;
+  }
+
+  .black-nav {
+    width: 1024px;
+    height: 40px;
+  }
+
+  .white-nav {
+    width: 1024px;
+    height: 48px;
+    background-color: rgb(247, 247, 247);
+  }
+
+  .right-nav {
+    width: 1024px;
+    height: 32px;
+  }
```

Comment:
Once you start styling for large screens (screens with width `1024px` and up), the `width` should be set to the full width of the screen (`width: 100%`). 

Here I have only marked the code for the `navbar`. On lines `681`, `686` and `692` change the width to `100%`.

Go through your CSS and change the width to 100% for screens with width 1024px and higher.